owners/src/owners.js

```diff
@@ -217,6 +218,16 @@ class OwnersRule {
   matchesFile(filePath) {
     return true;
   }
+
+  /**
+   * Describe the rule.
+   *
+   * @return {string} description of the rule.
+   */
+  toString() {
+    const ownersList = this.owners.length ? this.owners.join(', ') : '-';
```

Comment:
In what cases would this be empty? Only when the OWNERS.yaml file is empty? That's an error, right?

Reply:
You're right, I was thinking in cases where there are pattern-based rules but no regular rules, but in that case the rule would never be created without owners. Removing this logic.

Reply:
Done.